roles/letsencrypt/tasks/main.yml

```diff
@@ -125,7 +125,7 @@
     data: "{{ letsencrypt_challenge_response }}"
     dest: "{{ letsencrypt_cert_path }}"
     remaining_days: "{{ letsencrypt_remaining_days }}"
-  notify: restart apache
+  notify: "{{ letsencrypt_reload_handler }}"
```

Comment:
We could probably solve this by doing a generic notify topic of "restart web server" or something like that, and have both the apache and nginx handlers listen to that topic. But maybe that'll be weird if we wind up with both on the same host?